modules/ROOT/partials/commands/math-cmds.adoc

```diff
@@ -0,0 +1,11 @@
+[cols="1,3",options="header"]
+|===
+|Command |Description
+|Math |This command opens the {pluginname} dialog.
```

Comment:
```suggestion
|Math |This command opens the Math dialog.
```

Reply:
editor-command-identifiers.adoc uses this partial and does not know what `{pluginname}` is.

Reply:
![Screenshot 2024-05-02 at 6 05 13 am](https://github.com/tinymce/tinymce-docs/assets/96402951/781d97df-524b-46b4-93e8-802dbe3e4ca8)
@FarzadHayat if a variable is set in the header of the parent `.adoc` file such as the plugin.adoc file, any partials rendered inside the document will render as expected.